email/welcome-new-xccup.mjml

```diff
@@ -0,0 +1,85 @@
+      </mj-column>
+    </mj-section>
+    <mj-raw>
+
+      <!-- Intro text -->
+    </mj-raw>
+    <mj-section background-color="#fafafa">
+      <mj-column>
+        <mj-text font-size="20px" font-family="Helvetica Neue" color="#626262">Der neue XCCup</mj-text>
+        <mj-text color="#525252">
+          Liebe XCCup Pilot*innen
+          <p />
+
+          Im letzten Jahr hatte Wolf bereits angekündigt das unser XCCup Entwickler Sascha den Posten nach vielen Jahren gerne abgeben würde und ein Nachfolger gesucht wird.<br />
+
+          Diesem Aufruf sind wir (Kai & Stephan) gefolgt.
+          <p />
+          Da die aktuelle Webseite aus einer Zeit stammt in der Smartphones noch kein Thema waren ist die Bedienung auf mobilen Geräten eher umständlich gewesen. Aus diesem Grund haben wir den XCCup mit Fokus auf mobile Geräte von neu auf programmiert.
+          <br />
+          So gut wie alle Funktionen haben wir dabei übernommen und zusätzlich noch einige Neue hinzugefügt.<p />
+          Wir sind optimistisch, dass wir die neue Webseite zum Saisonstart “live” gehen lassen können. Bis dahin möchten wir euch bitten diese schon mal zu testen und uns zu kontaktieren, wenn etwas nicht wie erwartet funktionieren sollte. Am besten mit Screenshot und Angabe zum verwendeten Gerät und Browser.<br />
+          Ihr könnt dabei auch alte Flüge verwenden. Zu Testzwecken gibt es kein 14-Tages-Limit beim hochladen.<br />
+          Feedback bitte an $$EMAIL EINFÜGEN$$
+          <p />
+
+          Ihr findet die neue Version hier: <a href="https://next.xccup.net" target="_blank">https://next.xccup.net</a>
+          <p />
+          Zum login verwendet bitte eure E-Mail Adresse & euer aktuelles Passwort. Alle Daten werden nach der Testphase gelöscht und haben keine Auswirkungen auf die Wertung 2022. Probiert also gerne alles Mögliche aus.
+          <p />
+
+          <strong>Neue Funktionen:</strong>
+          <p />
+
+          - Man kann nun auf Kommentare antworten statt einfach nur einen Weiteren unter die Vorhandenen zu schreiben.
+          <p />
+          - Airbuddies: Solltet ihr zusammen mit anderen Piloten geflogen sein könnt ihr dessen Trackklog einblenden. Ihr müsst nichts weiter tun als euren Flug hoch zu laden und findet dann einen Button “Airbuddies” unter dem Barogramm. (Natürlich nur, wenn der andere Pilot auch seinen Fluch hochgeladen hat…)
+          <p />
+          - Die Tabellen der Wertungen zeigen abhängig von der Gerätegröße unterschiedlich viele Details an.
+          <p />
+          - Ihr könnt einen Hike&Fly Flug als solchen markieren. Die Höhenmeter werden in den Flugdetails angezeigt.
+          <p />
+          - Ihr könnt nun unterschiedliche Schirme im Profil hinterlegen und aus diesen beim hochladen auswählen.
+          <p />
+          - “Dark mode” abhängig von euren Geräteeinstellungen.
+          <p />
+          - $$ VORJAHRE ? $$
```

Comment:
Die Flüge der Vorjahre wurden im Großen und Ganzen importiert. Bitte prüft aber, ob ein für euch wichtiger Flug (z.B. persönlicher Rekord, ö.ä.) auch wirjlich vorhanden ist.

Reply:
Dafür müsste ich erst mal schauen, wie wir das implementieren. Bis jetzt kommt man ja nur über die Rekorde oder manuelles ersetzen der von 2022 in der URL an die alten.

Reply:
Ja dann sollen die Leute zumindest mal ihre Rekorde prüfen.

Wäre aufjedenfall mal wichtig.

Reply:
Vorjahre sind jetzt im anderen pr drin